R/get_debrief_doc_info.R

```diff
@@ -0,0 +1,20 @@
+#' Get debriefing doc info
+#'
+#' Get the specific information for the upcoming workshop so it can be used in the debriefing doc
+#'
+#' @param ws_dat information about the workshop to be inserted into the debriefing document
+#' @param meta_fld the URL of the meta folder
+#'
+#' @return debrief doc information to be used in rendering the doc
+#'
+#' @export
```

Comment:
Is it necessary for this function to be exported?
I can envision having a function that can be used by a user to check workshop information on the console. However, perhaps it would be nice to write a slightly more user friendly function, that would report the information to the console in a nicely readable format?

Reply:
thanks, may do this later!